applications/tari_base_node/src/recovery.rs

```diff
@@ -103,7 +103,7 @@ pub async fn run_recovery(node_config: &GlobalConfig) -> Result<(), anyhow::Erro
         pruning_horizon: node_config.pruning_horizon,
         pruning_interval: node_config.pruned_mode_cleanup_interval,
     };
-    let db = BlockchainDatabase::new(main_db, &rules, validators, db_config)?;
+    let db = BlockchainDatabase::new(main_db, &rules, validators, db_config, false)?;
```

Comment:
I would either set this as true, or make it configurable. 
Recovery mode will essentially strip and clean you normal db leaving it only with the blocks passing validation in the db. Following this methodology, I would not set it hard coded to false.

Reply:
Agreed, will change to `true`